### PR TITLE
feat(adapters): Add adapters for Rectangle, Angle and fix generate DICOM

### PR DESCRIPTION
**Pull request by wayfarer3130**
Add additional adapters for Rectangle, Angle, PlanarFreehand
Fix the generated DICOM so it included all the patient/study attributes so it doesn't fail to send against a validating PACS.  This is a critical bug in that the generated DICOM was missing the Patient ID attribute, and thus would fail to store on any DICOM compliant PACS back end.

### CODE REVIEW

File: packages/adapters/src/adapters/Cornerstone3D/MeasurementReport.ts [hunk 1]
```diff
@@ -132,25 +136,21 @@ export default class MeasurementReport {
         return _meta;
     }
 
-    static generateDerivationSourceDataset(
-        StudyInstanceUID,
-        SeriesInstanceUID
-    ) {
+    static generateDerivationSourceDataset = instance => {
```
Comment (wayfarer3130) [inline packages/adapters/src/adapters/Cornerstone3D/MeasurementReport.ts:139]: Use the actual instance data for the derived data, as it is used for generating the overall report.

File: packages/tools/src/tools/annotation/PlanarFreehandROITool.ts [hunk 2]
```diff
@@ -582,9 +582,11 @@ class PlanarFreehandROITool extends AnnotationTool {
     if (!(isDrawing || isEditingOpen || isEditingClosed)) {
       // No annotations are currently being modified, so we can just use the
       // render contour method to render all of them
-      annotations.forEach((annotation) =>
-        this.renderContour(enabledElement, svgDrawingHelper, annotation)
-      );
+      annotations.forEach((annotation) => {
+        console.log('annotation render', annotation);
```
Comment (sedghi) [inline packages/tools/src/tools/annotation/PlanarFreehandROITool.ts:586]: remove this log please

**Review by sedghi — CHANGES_REQUESTED** [review]
Please remove the unnecessarsy console log.

File: packages/adapters/rollup.config.mjs [hunk 3]
```diff
@@ -5,35 +5,38 @@ import commonjs from "@rollup/plugin-commonjs";
...
-    commonjs(),
-    typescript(),
-//    globals(),
-    // builtins(),
-    babel({
-      exclude: "node_modules/**"
-    }),
-    json()
-  ]
+    external: ["dcmjs"],
```
Comment (wayfarer3130) [inline packages/adapters/rollup.config.mjs:11]: Sorry about the refactor - the externals library is this line here - it also needs the globals, but I don't quite know how to figure out the global name for the other externals.

**Review by sedghi — APPROVED** [review]

**Pull request merged**